README.md

```diff
@@ -153,6 +153,12 @@ duplicate([1,2,3,4,5]); // [1,2,3,4,5,1,2,3,4,5]
 * Explain the differences on the usage of `foo` between `function foo() {}` and `var foo = function() {}`
 * What are the differences between variables created using `let`, `var` or `const`?
 * What are the differences between ES6 class and ES5 function constructors?
+* What are some advantages for using the new arrow `=>` function syntax?
```

Comment:
I'd rather ask for the best use cases, instead of the *advantages*.

Reply:
I am picking up what you are putting down.